src/cpp/DQM/VisDQMRenderTools.h

```diff
@@ -375,6 +376,11 @@ static string statWithErrorToJson(const TH2* const hist, const char* const name)
       .arg(name);
 }
 
+static string rootObjectToArray(const TObject* const h)
```

Comment:
I think I might used some other function definition as reference and accidentally left array part of name unchanged. Changed name now.